product_docs/docs/pgd/5/consistency/conflicts.mdx

```diff
@@ -674,9 +674,16 @@ This matrix helps you individuate the conflict types the conflict resolvers can
 | delete_missing            | skip                     |
 | target_column_missing     | ignore_if_null           |
 | source_column_missing     | use_default_value        |
-| target_table_missing      | skip_if_recently_dropped |
+| target_table_missing (see note)     | skip_if_recently_dropped |
 | apply_error_ddl           | error                    |
 
+<div id="target_table_note"></div>
+
+!!! note target_table_missing
+This conflict type is not detectable on Community Postgresql and if the target table is missing, it will error and halt replication. 
```

Comment:
Thank you for highlighting this! Minor nitpick:

```suggestion
This conflict type is not detectable on Community Postgresql, and if the target table is missing, it will cause an error and halt replication.
```